project-docs/javascript-linter-spec.md

```diff
@@ -0,0 +1,163 @@
+# JavaScript reference docs structure specification
+
+This document describes the structure we want to require for JavaScript reference documentation.
+
+## Recipes and ingredients overview
+
+It is expected that pages documenting different sorts of thing will need to contain different elements. For example, the page for the JavaScript `BigInt` class will want to contain lists of links to its members, while the page for the JavaScript `BigInt.asUintN()` static method will want to contain a section describing the method syntax.
+
+To reflect this, a "recipe" is a short YAML file that describes at a high level the elements that a particular type of page is expected to have. A recipe provides a list of elements, that are expected to appear in the page in the order given. So for each distinct type of page, we define a single recipe.
```

Comment:
>  I'm not asking you to do this unless you're a little excited about it.

How much is a little ? :)

I'm happy to have a go at this but think it will be less confusing if we get through the substantive comments first.

Reply:
-> https://github.com/mdn/stumptown-content/pull/391/commits/27178729dd1d32f302a664348cf2e5a8e8de756e

I made some minor changes here but didn't think it was worth going for some full-scale rewrite, especially as we have no actual template for requirements docs, so I would really be guessing about the exact language to use.